updated get message api (#75)
postalserver/docs: content/4.developer/1.api.md

  }
}
```

## GET Message

> To retrieve a message and its contents, use the `GET` method with the `id` and `_expansions` parameters for the message from Postal. For more details, refer to the [Postal API documentation](https://apiv1.postalserver.io).

Reviewer: ```suggestion
To retrieve a message and its contents, use the `GET` method with the `id` (received when sending the message) and `_expansions` parameters (if you need more information than the basics) for the message from Postal. For more details, refer to the [Postal API documentation](https://postalserver.github.io/postal-api/controllers/messages/message.html).
```

> #### Request Format
> 
> ```json

Reviewer: ```suggestion
```json
```

> POST https://<your.postal.domain.com>/api/v1/messages/message
> {
>   "id": 14,    ## Provide the message ID after you start receiving emails. Use `1` if you're testing or unsure.

Reviewer: ```suggestion
{
  "id": 14,
```

  "_expansions": true
}
```

### Example cURL Request

You can use the following cURL command to make the request:

```bash
curl --location 'https://<your.postal.domain.com>/api/v1/messages/message' \

> --header 'X-Server-API-Key: <ClxzMBRbNLkHqQABmuta3Amx>' \

Reviewer: ```suggestion
--header 'X-Server-API-Key: $yourAPIKeyFromTheCredentialsPage' \
```
